.github/workflows/toolchain-version-check.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name: Toolchain Version Check
+"on":
+  schedule:
+    - cron: "0 0 * * *" # build nightly!
```

Comment:
Rust releases go out every 6 weeks on Thursdays. We can probably limit this to run once a week on Fridays, maybe at 5am UTC to avoid racing with the nightly release job?